packages/create-plugin/src/utils/utils.templates.ts

```diff
@@ -4,7 +4,7 @@ import fs from 'node:fs';
 import { mkdirp } from 'mkdirp';
 import createDebug from 'debug';
 import { filterOutCommonFiles, isFile, isFileStartingWith } from './utils.files.js';
-import { renderHandlebarsTemplate } from './utils.handlebars.js';
+import { renderHandlebarsTemplate, normalizeId } from './utils.handlebars.js';
```

Comment:
I think we should move normalizeId out of this file and delete the normalize_id helper entirely to prevent future changes reintroducing it to the templates.

Reply:
Let's not do this in this PR. It's been tackled [in this PR](https://github.com/grafana/plugin-tools/pull/834).